PWGHF/D2H/Tasks/taskXic.cxx

```diff
@@ -34,172 +34,275 @@ using namespace o2::analysis::hf_cuts_xic_to_p_k_pi;
 struct HfTaskXic {
   Configurable<int> selectionFlagXic{"selectionFlagXic", 1, "Selection Flag for Xic"};
   Configurable<double> yCandMax{"yCandMax", -1., "max. cand. rapidity"};
+  Configurable<float> trackEtaCut{"trackEtaCut", 4.0, "eta cuts for tracks"};
+  Configurable<float> trackDcaXYCut{"trackDcaXYCut", 0.0025, "DCAxy cuts for tracks"};
+  Configurable<float> trackDcaZCut{"trackDcaZCut", 0.0025, "DCAz cuts for tracks"};
```

Comment:
Please, don't use `Cut`. Instead use `Min`, `Max` to clearly indicate the type of criterion and mention it in the description.
Also use the recommended order: `etaTrack`, `dcaXYTrack`,...